packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx

```diff
@@ -77,6 +81,7 @@ function Root({
   logoHref = 'https://github.com',
   title,
   titleHref = '/',
+  onMobileMenuToggle,
```

Comment:
Can you change this to `onNarrowMenuToggle`. We try to avoid using device-terms in our APIs.